conf/app.routes

```diff
@@ -3,4 +3,7 @@
 POST                    /movements/arrivals                 controllers.MovementsController.createMovement
 GET                     /movements/arrivals                 controllers.MovementsController.getArrivals
 
+GET                    /movements/arrivals/:arrivalId                 controllers.MovementsController.get(arrivalId: ArrivalId)
+PUT                    /movements/arrivals/:arrivalId                 controllers.MovementsController.updateArrival(arrivalId: ArrivalId)
```

Comment:
This is going to give us back a different arrival each time it's called, is that right? I know POST vs PUT debate can be a bit hairy so I'll tread carefully here ;D I understand having a POST to handle every request to send a message would require createMovement to handle the internals of adding or creating a message. Is that a bad thing? (you've probably all discussed this to death)